Simplified.xcodeproj/project.pbxproj

```diff
@@ -2830,7 +2892,7 @@
 					"$(inherited)",
 				);
 				INFOPLIST_FILE = "Simplified/Simplified-Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.3;
```

Comment:
The reason we bump the version to 10.0 is because we are using `SecKeyCreateWithData` to create SecKey when we extract the PEM data from Keychain, in NYPLAudiobookToolkit. 
Unless we figure out how to create the Seckey without using this API, otherwise we need to keep the version at 10.0

Reply:
I see, changed back